cognee/infrastructure/databases/vector/embeddings/OllamaEmbeddingEngine.py

```
@@ -71,13 +69,13 @@ async def _get_embedding(self, client: httpx.AsyncClient, prompt: str) -> List[f
             try:
-                response = await client.post(
-                    self.endpoint, json=payload, headers=headers, timeout=60.0
-                )
-                response.raise_for_status()
-                data = response.json()
-                return data["embedding"]
-            except httpx.HTTPStatusError as e:
+                async with aiohttp.ClientSession() as session:
+                    async with session.get(
+                        self.endpoint, json=payload, headers=headers, timeout=60.0
+                    ) as response:
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)